airbyte-webapp/src/test-utils/testutils.tsx

```diff
@@ -31,15 +32,17 @@ export async function render<
     return (
       <TestWrapper>
         <ConfigContext.Provider value={{ config: defaultConfig }}>
-          <FeatureService features={defaultFeatures}>
-            <ServicesProvider>
-              <QueryClientProvider client={queryClient}>
-                <MemoryRouter>
-                  <Suspense fallback={<div>'fallback content'</div>}>{children}</Suspense>
-                </MemoryRouter>
-              </QueryClientProvider>
-            </ServicesProvider>
-          </FeatureService>
+          <AnalyticsProvider>
+            <FeatureService features={defaultFeatures}>
+              <ServicesProvider>
+                <QueryClientProvider client={queryClient}>
+                  <MemoryRouter>
+                    <Suspense fallback={<div>'fallback content'</div>}>{children}</Suspense>
+                  </MemoryRouter>
+                </QueryClientProvider>
+              </ServicesProvider>
+            </FeatureService>
+          </AnalyticsProvider>
```

Comment:
This was added and removed due to me misunderstanding a failing test. But it needs to be here for my tests.